editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -26,17 +27,22 @@ const CATEGORY_COLORS: Record<string, string> = {
   Flooding: "#ECECEC",
 };
 
-interface StyledConstraintProps extends BoxProps {
+const PREFIX = "ConstraintsList";
+
+const classes = {
+  content: `${PREFIX}-content`
+}
+
+interface StyledAccordionProps extends BoxProps {
   category: string;
 }
 
-const StyledConstraint = styled(Box, {
-  shouldForwardProp: (prop) => prop !== "category",
-})<StyledConstraintProps>(({ theme, category }) => ({
+const StyledAccordion = styled(Accordion, {
+  shouldForwardProp: (prop) => !["category", "metadata", "content", "data"].includes(prop as string),
```

Comment:
Verbose, but it resolves this issue - 

<img width="540" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/12a65706-40af-4444-9116-6b322f33d98e">